test/e2e/framework.go

```diff
@@ -669,6 +671,54 @@ func (data *TestData) deleteService(name string) error {
 	return nil
 }
 
+// createNetworkPolicyToTestIPBlockWithExcept creates a network policy with IPBlock's Except field.
+func (data *TestData) createNetworkPolicyToTestIPBlockWithExcept(name string, exceptIP string) (*networkingv1.NetworkPolicy, error) {
```

Comment:
I feel like this is a very specific function and it would probably be better to put it directly in networkpolicy_test.go

Reply:
also there is no real reason to have `ToTest` in the function name

Reply:
Renamed the function and moved it to networkpolicy_test.go.